README.md

```diff
@@ -91,7 +88,9 @@ Traefik routes requests based on host name. which includes the coinstack name. F
 - Start a coinstack:
 
   ```sh
-  cd node/coinstacks/ethereum && docker-compose up
+  cd node/coinstacks/ethereum
+  cp sample.env .env // make sure to populate the .env file with valid API keys if required
```

Comment:
this step is detailed in the node/README.md, though probably doesn't hurt to have in both places, this was just a more direct/succinct running example.

Reply:
I'd either keep it in here or just put the link here to one of the coin-specific readme's so that it's possible to follow a set of explicit steps

Reply:
lets keep it